snappy/click.go

```diff
@@ -411,13 +426,17 @@ func (m *packageYaml) removePackageServices(baseDir string, inter interacter) er
 	return nil
 }
 
-func (m *packageYaml) addPackageBinaries(baseDir string) error {
+func (m *snapYaml) addPackageBinaries(baseDir string) error {
 	if err := os.MkdirAll(dirs.SnapBinariesDir, 0755); err != nil {
 		return err
 	}
 
-	for _, binary := range m.Binaries {
-		aaProfile, err := getSecurityProfile(m, binary.Name, baseDir)
+	for _, app := range m.Apps {
+		if app.Daemon != "" {
+			continue
+		}
+
+		aaProfile, err := getSecurityProfile(m, app.Name, baseDir)
```

Comment:
More intersection with skills (again, this is not a blocking comment).

How do we use skills to generate _all_ security files? How do we marry this with the state engine and transactionality?